### PR TITLE
add new entries and translations

### PR DESCRIPTION
**Pull request by stardustichi**

### CODE REVIEW

File: src/data/missions.csv [hunk 1]
```diff
@@ -1298,14 +1303,14 @@
...
-"【宝塚記念】育成を10回完了しよう",""
-"【宝塚記念】育成で期間中にファン数を50万人獲得しよう",""
-"【宝塚記念】育成で期間中にファン数を150万人獲得しよう",""
-"【宝塚記念】育成で宝塚記念に勝利しよう",""
-"【宝塚記念】限定ミッションをすべてクリアしよう",""
+"【宝塚記念】育成を1回完了しよう","[Takarazuka Kinen] Complete a training once"
+"【宝塚記念】育成を3回完了しよう","[Takarazuka Kinen] Complete a training 3 times"
+"【宝塚記念】育成を5回完了しよう","[Takarazuka Kinen] Complete a training 5 times"
+"【宝塚記念】育成を10回完了しよう","[Takarazuka Kinen] Complete a training 10 times"
+"【宝塚記念】育成で期間中にファン数を50万人獲得しよう","[Takarazuka Kinen] Accumulate 0.5 million fans during the period"
```
Comment (latoulicious) [inline src/data/missions.csv:1310]: Just asking. Why didn't use 500k instead 0.5 Mil? text spacing saving purpose?

Comment (stardustichi) [reply src/data/missions.csv:1310]: i guess im just trying to keep 万 consistent, but if others want me to change that ill change it

File: src/data/support-title.csv [hunk 2]
```diff
@@ -65,4 +66,6 @@
 "[花嫁たるもの！！]","[What Makes a Bride!!]"
 "[召しませふぁーすとBite！]","[Try First with Bite!]"
 "[おもい、ねがう]","[A Feeling, a Wish]"
+"[Head-on fight！]","[Head-on fight!]"
 "[ふたつのノーブルライト]","[Two Noble Lights]"
+"[祝福はフーガ]","[Blessing Is a Fugue]"
```
Comment (FabulousCupcake) [inline src/data/support-title.csv:71]: @robflop wdyt?

File: src/data/missions.csv (hunk 1 above)
Comment (robflop) [reply src/data/missions.csv:1310]: I use 500 thousand or such in the mission translations of the master doc, personally.

File: src/data/support-title.csv (hunk 2 above)
Comment (robflop) [reply src/data/support-title.csv:71]: I'd add an "a", i.e. "A Blessing Is a Fugue", since otherwise this seems a bit chopped off. 
Alternatively, "Blessings are Fugues", since singular/plural isn't reflected in JP, so it could be either.

2nd Variant with plural sounds better here, i think.

File: src/data/uma-nickname-requirements.csv [hunk 3]
```diff
@@ -77,6 +77,7 @@
 "メジロの冠名がつくウマ娘にGⅠで1回以上勝利し、\n菊花賞、宝塚記念、有馬記念で勝利して、\nファン数が240000人以上になる","<size=22>Win a GI race against a Mejiro at least once\nWin the Kikka Sho, Takarazuka Kinen, and Arima Kinen\nEarn a total of ≥240000 fans</size>"
 "ユニコーンS、安田記念を含むマイルの重賞で5勝以上し、\nうち1勝以上は天候（雨）で勝利かつ、\nマイルCSを5バ身差以上で勝利する","<size=22>Win a big race ≥5 times including the Unicorn Stakes, and Yasuda Kinen with one of them in rainy weather\nWin the Mile Championship with difference of ≥5 lengths</size>"
 "GⅠを1番人気で6回以上出走し、\nマイルCS、安田記念、有馬記念で勝利して\nファン数が240000人以上になる","<size=22>Participate a GI race ≥6 times being the 1st popular\nWin the Mile Championship, Yasuda Kinen, and Arima Kinen\nEarn a total of ≥240000 fans</size>"
+"作戦「追込」かつ1番人気で重賞6連勝以上、\nうち2戦以上はラストの直線で5人以上を抜いて勝利する","<size=22>As a chaser being the number one popular, win ≥6 consecutive big races,\nwin ≥2 of them by overtaking ≥5 other runners on the last straights</size>"
```
Comment (robflop) [inline src/data/uma-nickname-requirements.csv:80]: I'd reword the last bit of this if possible space-wise to read:

"[...] win ≥6 consecutive big races, with overtaking ≥5 other runners on the last straight for ≥2 of them"

File: src/data/missions.csv [hunk 4]
```diff
@@ -1298,14 +1303,14 @@
...
-"【宝塚記念】育成を10回完了しよう",""
-"【宝塚記念】育成で期間中にファン数を50万人獲得しよう",""
-"【宝塚記念】育成で期間中にファン数を150万人獲得しよう",""
-"【宝塚記念】育成で宝塚記念に勝利しよう",""
-"【宝塚記念】限定ミッションをすべてクリアしよう",""
+"【宝塚記念】育成を1回完了しよう","[Takarazuka Kinen] Complete a training once"
+"【宝塚記念】育成を3回完了しよう","[Takarazuka Kinen] Complete a training 3 times"
+"【宝塚記念】育成を5回完了しよう","[Takarazuka Kinen] Complete a training 5 times"
+"【宝塚記念】育成を10回完了しよう","[Takarazuka Kinen] Complete a training 10 times"
+"【宝塚記念】育成で期間中にファン数を50万人獲得しよう","[Takarazuka Kinen] Accumulate 500k fans during the period"
```
Comment (robflop) [inline src/data/missions.csv:1310]: During "the" period doesn't really give you any info (how long is "the" period? etc), so i'd change this to read "the mission period".

**Review by robflop — CHANGES_REQUESTED** [review]

File: src/data/skill-desc.csv [hunk 5]
```diff
@@ -25,6 +25,7 @@
 "レース後半で\n1番をキープし続ける力を発揮する","<size=22>Display your power in keeping the lead position during the latter half of the race</size>"
 "最終コーナーで好位置にいると\n弾みを付けて加速力が上がる","<size=22>Be in a good position at the last corner to gain momentum and increase your acceleration</size>"
 "最終直線で後ろの方から追い抜くと\n末脚を使って速度が上がる","<size=22>Increase speed when passing in the back in Final Leg straight</size>"
+"最終直線で後ろの方から外を回って\n追い抜く時、速度を上げる","<size=22>On last straight, gain speed when passing from the outside back</size>"
```
Comment (robflop) [inline src/data/skill-desc.csv:28]: Add a "the" here, i.e. "On the left straight"

File: src/data/skill-desc.csv [hunk 6]
```diff
@@ -386,6 +387,7 @@
 "レース後半で1番を\nキープし続ける力をちょっと発揮する","<size=22>Display a bit of your ability to keep 1st place during the later half of the race</size>"
 "最終コーナーで好位置にいると\n加速力がわずかに上がる","<size=22>Be in good position at last corner to slightly increase acceleration</size>"
 "最終直線で後ろの方から追い抜くと\n速度がわずかに上がる","<size=22>Your speed increases slightly when passing during the last straight</size>"
+"最終直線で後ろの方から外を回って\n追い抜く時、速度をわずかに上げる","<size=22>On last straight, gain slight speed when passing from the outside back</size>"
```
Comment (robflop) [inline src/data/skill-desc.csv:390]: Same here, add a "the" to the start, i.e. "On the last straight"

File: src/data/skill-desc.csv (hunk 5 above)
Comment (stardustichi) [reply src/data/skill-desc.csv:28]: yeah im cutting them to prevent the text from being too long

Comment (robflop) [reply src/data/skill-desc.csv:28]: Does it really not fit? Some other texts are longer, so adding a "the" there shouldn't be a big problem. Though if it is then yea ofc, leave it out.

Comment (stardustichi) [reply src/data/skill-desc.csv:28]: well it is as you said wouldnt be a problem, but then i again that means i need to add "the" for the other similar cases lol

Comment (robflop) [reply src/data/skill-desc.csv:28]: Guess it could stay like that then, and be included in a consistency PR if anyone wants to make that.

**Review by robflop — APPROVED** [review]
